tracker/index.js

```diff
@@ -99,9 +78,40 @@ import { removeTrailingSlash } from '../lib/url';
     });
   };
 
-  /* Start */
+  if (!window.umamiTrack) {
+    window.umamiTrack = (type, params, id) => {
```

Comment:
I'd rather see these as methods on the `window.umami` object instead of it's own thing. I think the pageviews should take `url` and `referrer` while events should take `url`, `event_type` and `event_value` and they can both roll up to a parent `collect` function.

Reply:
Check. Changed everything according to your suggestions.

`window.umami('event_value') // event type is "custom" ` so it works the same
additionally we have 
`window.umami.pageView() // with url and ref as optional params`
`window.umami.event('event_type', 'event_value') // also used by auto events`